doc/content/the-things-stack/monitor/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Monitoring in The Things Stack"
```

Comment:
I think it's better if we just add `Alerting` as the top level instead of this one. We can move it out later when I create the `features` section.